src/MegaPoker.sol

```diff
@@ -18,104 +18,91 @@
 
 pragma solidity ^0.6.11;
 
-interface OsmLike {
-    function poke() external;
-    function pass() external view returns (bool);
+contract Addresses {
```

Comment:
Naming suggestion: `OsmAddresses` to reduce the potential for confusion.

Reply:
Will change it!

Reply:
Actually it has also the spotter address so maybe `ContractAddresses`?

Reply:
or `PokingAddresses`

Reply:
Done!